### PR TITLE
Include pylinter and make code compatible, codeql findings

### PR DESCRIPTION
**Pull request by m-1-k-3**
* included a pylinter check in check_project.sh
* started with some first cleanup tasks to make the code pylinter compatible
  started rating: 5.58/10
  current rating: 9.52/10
* CodeQL issues are fixed
* as the docker mode of emba is currently not supported we have disabled it in the UI
* SECRET_KEY is moved to .env and auto-generated from installer script (old and insecure default key is used for build scripts)
* web.log is not needed anymore for build checks
* Moved redis to port 7777 (we do not conflict with cve-search anymore)
* if empty database show "no data" in mainDashboard
* emba command is now included in the overview of a scan

### CODE REVIEW

File: embark/templates/uploader/individualReportDashboard.html [hunk 1]
```diff
@@ -1,6 +1,13 @@
 {% extends "uploader/home.html" %}
 {% block maincontent %}
 
+<style>
+tbody tr td:first-child {
+  float: left;
+  min-width: 350px;
+}
+</style>
```
Comment (p4cx) [inline embark/templates/uploader/individualReportDashboard.html]: Better: Add a class/id to the table, to which the style should be attached and move this code into one of the imported css files.

File: embark/uploader/views.py [hunk 2]
```diff
@@ -318,6 +318,7 @@ def html_report(request, analyze_id, hmtl_file):
     report_path = Path(f'/app/emba{request.path}')
 
     html_body = get_template(report_path)
+    logger.info("html_report - analyze_id: %s hmtl_file: %s", analyze_id, hmtl_file)
```
Comment (p4cx) [inline embark/uploader/views.py]: typo

Comment (m-1-k-3) [reply embark/uploader/views.py]: hmtl_file? It is also in the function def ... so, I have not changed it for now.

**Review by p4cx — COMMENTED** [review]
Still have to test it, but as far as I can see it looks good. Will approve if test runs :)

File: embark/uploader/views.py (hunk 2 above)
Comment (p4cx) [reply embark/uploader/views.py]: I think it is a typo and both should be renamed to `html_file`.

**Review by p4cx — APPROVED** [review]

**Pull request merged**